src/Metapackage/Metapackage.csproj

```diff
@@ -7,7 +7,15 @@
     <ProjectName>Cythral.CloudFormation.CustomResource</ProjectName>
     <AssemblyName>$(ProjectName)</AssemblyName>
     <PackageId>$(ProjectName)</PackageId>
-  </PropertyGroup>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
```

Comment:
The metapackage will not benefit from SourceLink since it only brings in other dependencies - by itself it does not contain any source code